src/types.rs

```diff
@@ -860,7 +919,9 @@ pub struct MessageStandardAttachment {
     pub thumb_url: Option<String>,
     pub title: Option<String>,
     pub title_link: Option<String>,
-    pub ts: Option<f32>,
+    #[serde(deserialize_with = "deserialize_timestamp")]
+    #[serde(default)]
```

Comment:
Do we need this? I believe this just going to set `None` anyway since that's the `Default` impl for `Option`.

Reply:
We do not. That was left over from when I was trying to figure out how to deserialize into an Option. Oops.

Reply:
I thought I replied to this, but I don't see anything here.

This is necessary, per testing. Some instances of this struct will fail to deserialize without it. I think that's because the custom deserializer does/has to return an Option<Timestamp>; the defaulting would be  handled if this were derived but it isn't.